lib/dialects/cockroachdb.ts

```diff
@@ -67,17 +66,12 @@ export function rawColumnToColumn(rawColumn: RawColumn): Column {
  * Eg `'example'::character varying` => `example`
  */
 export function parseDefaultValue(type: string | null) {
-  if (isNil(type)) return null;
```

Comment:
That's a whole rabbit hole I don't want to get into in this ticket. The type checking will fail before this function gets hit since `rawColumnToColumn` doesn't correctly handle null or empty responses from Knex. But assuming we get success from Knex, the column result should be `string | null` so I think this makes sense. Otherwise we should update the function signature.

Reply:
I don't recall ever seeing an `undefined` coming from the database on purpose. The only case the default value would be undefined is if we made a mistake in the SQL that constructs the object properties in the first place 👍🏻